### DISCUSSION TITLE
Error Notification

### DISCUSSION THREAD

**Discussion by guala75** [elk-zone/elk#3076] [Q&A]
Good afternoon everyone!!!

in Elk con Mastodon after blocking a server domain this string appeared in my notifications!!!

![bug_1](https://github.com/user-attachments/assets/921c35cc-ac23-41c9-ab1f-d1c8d4e9d060)

in the original webclient it doesn't happen...
![bug_2](https://github.com/user-attachments/assets/1ef1acc4-c9c8-40a4-abeb-68197a231b7e)

what could it depend on?

**Comment by shuuji3** [comment]
Hello!🙂 This notification type `severed_relationships` was introduced by the recent Mastodon release v4.3 on October 2024 (ref. https://docs.joinmastodon.org/entities/Notification/#type), so Elk hasn't implemented this new type yet, resulting in the unexpected rendering.

Elk needs to be implemented to support this notification type. I created a new feature request Issue for that: https://github.com/elk-zone/elk/issues/3077. Please follow the issue if you'd like to receive updates.
